browser/src/control/Control.MobileTopBar.js

```diff
@@ -71,7 +72,17 @@ L.Control.MobileTopBar = L.Control.extend({
 				{type: 'toolitem',  id: 'redo', text: _UNO('.uno:Redo'), command: '.uno:Redo', enabled: false},
 				{type: 'customtoolitem', visible: false, id: 'acceptformula', command: 'acceptformula', text: _('Accept')},
 				{type: 'customtoolitem', visible: false, id: 'cancelformula', command: 'cancelformula', text: _('Cancel')},
-				{type: 'spacer'},
+				{type: 'spacer', id: 'before-PermissionMode'},
+				{
+					type: 'container',
+					id: 'permissionmode-container',
+					children: [
+						{type: 'htmlcontent', id: 'PermissionMode', htmlId: 'permissionmode', text: '&nbsp;&nbsp;&nbsp;&nbsp;&nbsp', isReadOnlyMode: isReadOnlyMode, canUserWrite: canUserWrite, visible: false},
+						{type: 'spacer', id: 'after-PermissionMode', visible: false},
+					],
+					vertical: false,
+				},
+				{type: 'spacer', id: 'after-PermissionMode'},
```

Comment:
dup

Reply:
instead of copy pasting the same item, maybe please create function for it, similar like it is done in: https://github.com/CollaboraOnline/online/blob/master/browser/src/control/Control.StatusBar.js#L193